devito/__init__.py

```diff
@@ -109,3 +126,26 @@ def _switch_cpu(develop_mode):
 
 # Expose a mechanism to clean up the symbol caches (SymPy's, Devito's)
 clear_cache = CacheManager().clear  # noqa
+
+
+# Helper functions to switch on/off optimisation levels
+def mode_develop():
```

Comment:
note for reviewers: these three functions have just been moved from parameters.py